tests/store/test_sftp_artifact_repo.py

```diff
@@ -13,6 +13,7 @@ def sftp_mock():
     return MagicMock(autospec=pysftp.Connection)
 
 
+@pytest.mark.large
```

Comment:
Q: Why the large annotation here? Seems fine to add, but was this test slow beforehand?

Reply:
It takes about 60 seconds to throw the error.